packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -220,7 +218,6 @@ const AccountSignupModal = ({
             <AccountSignup
                 onSignup={onSignup}
                 residence_list={residence_list}
-                is_eu={is_eu}
```

Comment:
[Clarification] Is `is_eu` no longer required in account sign up?

Reply:
Yes. It should be removed 👍🏻

Reply:
`is_eu` wasn't used here before, so we just remove it.